firebase-vscode/webviews/globals/app.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { ReactNode, StrictMode } from "react";
```

Comment:
Nit: here and with extension-state.tsx - I think it's a convention to name all React components Pascal case although I know there is an edge case where some OS's have some issues with Pascal case, but we should probably have a consistent naming convention. So if you're concerned about that we could convert the existing components to snake case.

Reply:
ack - yeah, I'm on the fence here... since this file is exporting a component and a hook, does the component take precedence? also, funny you mentioned snake-case, because g3 is very much a snake-case environment, but it seems like this repo is more of a hyphen-env?

Reply:
Going to keep as-is, but will give this some more thought and try to come up with concrete style-guide on when to use kebab vs pascal.